kusto/conn.go

```diff
@@ -54,12 +57,15 @@ func newConn(endpoint string, auth Authorization, client *http.Client) (*conn, e
 		return nil, errors.ES(errors.OpServConn, errors.KClientArgs, "could not parse the endpoint(%s): %s", endpoint, err).SetNoRetry()
 	}
 
-	c := &conn{
-		auth:        auth.Authorizer,
-		endMgmt:     &url.URL{Scheme: "https", Host: u.Host, Path: "/v1/rest/mgmt"},
-		endQuery:    &url.URL{Scheme: "https", Host: u.Host, Path: "/v2/rest/query"},
-		streamQuery: &url.URL{Scheme: "https", Host: u.Host, Path: "/v1/rest/ingest/"},
-		client:      client,
+	c.endMgmt = &url.URL{Scheme: "https", Host: u.Host, Path: "/v1/rest/mgmt"}
```

Comment:
This and the code above are identical except for the new fields added. That can be left nil/blank right ?